### PR TITLE
[Docs] Improve doc structure

### PR DESCRIPTION
**Pull request by jillnogold**
update demos section, delete duplicated topics, rearrange topics, add links

### CODE REVIEW

**Review by alonmr — APPROVED** [review]

File: docs/tutorials/index.md [hunk 1]
```diff
@@ -3,8 +3,7 @@
 
 The following tutorials provide a hands-on introduction to using MLRun to implement a data science workflow and automate machine-learning operations (MLOps).
 
-- [**Quick-start Tutorial**](./01-mlrun-basics.html) 
-- [**Gen AI tutorials**](#gen-ai-tutorials)
+- [**Gen AI tutorials**](#gen--ai-tutorials)
```
Comment (rokatyy) [inline docs/tutorials/index.md]: why? I see that in the latest documentation this links works  - https://docs.mlrun.org/en/latest/tutorials/index.html#gen-ai-tutorials

File: docs/tutorials/demos.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ For full usage instructions, run the script with the `-h` or `--help` flag: ```!
...
-|<b>[Fraud Prevention (Feature Store)](https://github.com/mlrun/demos/tree/master/fraud-prevention-feature-store)</b>|This demo shows the usage of MLRun and the feature store. Fraud prevention specifically is a challenge as it requires processing raw transaction and events in real-time and being able to quickly respond and block transactions before they occur. Consider, for example, a case where you would like to evaluate the average transaction amount. When training the model, it is common to take a DataFrame and just calculate the average. However, when dealing with real-time/online scenarios, this average has to be calculated incrementally.|
+|<b>[Mask Detection Demo](https://https://github.com/mlrun/demo-mask-detection)</b>|This demo contains three notebooks that: Serve the model as a serverless function in an http endpoint; Train and evaluate a model for detecting if an image includes a person who is wearing a mask, by using Tensorflow, Keras, or PyTorch; Write an automatic pipeline where you download a dataset of images, train and evaluate the model, then optimize the model (using ONNX) and serve it.|
```
Comment (alonmr) [inline docs/tutorials/demos.md]:
```suggestion
|<b>[Mask Detection Demo](https://github.com/mlrun/demo-mask-detection)</b>|This demo contains three notebooks that: Serve the model as a serverless function in an http endpoint; Train and evaluate a model for detecting if an image includes a person who is wearing a mask, by using Tensorflow, Keras, or PyTorch; Write an automatic pipeline where you download a dataset of images, train and evaluate the model, then optimize the model (using ONNX) and serve it.|
```

File: docs/tutorials/demos.md [hunk 3]
```diff
@@ -29,8 +29,8 @@ For full usage instructions, run the script with the `-h` or `--help` flag: ```!
...
+|<b>[Fraud Prevention (Feature Store)](https://https://github.com/mlrun/demo-fraud)</b>|This demo shows the usage of MLRun and the feature store. Fraud prevention specifically is a challenge as it requires processing raw transaction and events in real-time and being able to quickly respond and block transactions before they occur. Consider, for example, a case where you would like to evaluate the average transaction amount. When training the model, it is common to take a DataFrame and just calculate the average. However, when dealing with real-time/online scenarios, this average has to be calculated incrementally.|
```
Comment (alonmr) [inline docs/tutorials/demos.md]:
```suggestion
|<b>[Fraud Prevention (Feature Store)](https://github.com/mlrun/demo-fraud)</b>|This demo shows the usage of MLRun and the feature store. Fraud prevention specifically is a challenge as it requires processing raw transaction and events in real-time and being able to quickly respond and block transactions before they occur. Consider, for example, a case where you would like to evaluate the average transaction amount. When training the model, it is common to take a DataFrame and just calculate the average. However, when dealing with real-time/online scenarios, this average has to be calculated incrementally.|
```

**Review by alonmr — APPROVED** [review]
